src/linagora.esn.calendar/app/calendar-configuration/calendar-configuration-tab-main/calendar-configuration-tab-main.controller.js

```diff
@@ -53,12 +53,11 @@ require('../../components/modals/calendar-delete-confirmation/calendar-delete-co
       self.isResource = self.calendar.type === CAL_CALENDAR_TYPE.RESOURCE;
 
       if (!self.newCalendar && self.calendar) {
-        calCalDAVURLService.getCalendarURL(self.calendar).then(function(url) {
-          self.caldavurl = url;
```

Comment:
What's the purpose of this line in the old code?

Reply:
nothing actually. the value was fetched and that's it.
and when building the export URL the old code assumed you will use the ESN backend.

Reply:
OK